dist/carousel/carousel.css

```diff
@@ -234,14 +234,8 @@ span.carousel__container {
     padding-bottom: 10px;
     scroll-behavior: smooth;
     /* stylelint-disable declaration-block-no-duplicate-properties */
-    -webkit-scroll-snap-type: mandatory;
-    -webkit-scroll-snap-type: x mandatory;
-    -ms-scroll-snap-type: mandatory;
-    -ms-scroll-snap-type: x mandatory;
     scroll-snap-type: proximity;
-    -webkit-scroll-snap-type: x proximity;
-        -ms-scroll-snap-type: x proximity;
-            scroll-snap-type: x proximity;
+    scroll-snap-type: x proximity;
```

Comment:
Hmm this shouldn't be changing. I've seen this section removed again (Luke added it back in one of my PRs). I think you might need to rebuild with new node modules.

Reply:
> Hmm this shouldn't be changing. I've seen this section removed again (Luke added it back in one of my PRs). I think you might need to rebuild with new node modules.

I rebased from `16.0.0`, cleaned and did an `npm i` and ...I don't see any diffs. Is it possible support for the property got good enough that the vendor prefixes are no longer needed? I see those vendor prefixes exist in the `.less` file also. Maybe the autoprefixer is removing them intentionally? 
<img width="1364" alt="image" src="https://user-images.githubusercontent.com/1675667/226933837-5bd2a6b6-a0a8-4d6e-b660-7089f69b445b.png">

Reply:
So this happened to me as well. But when my PR for icons was up, those were reverted. 
https://github.com/eBay/skin/pull/2006/commits/8edf506039efe1263e71af23edec400877238a43

So we need to pick a way and stick with it and make sure it doesn't get reverted.

Reply:
Maybe your PR for icons branch did not have the latest `node-modules`? idk. I'm fine either way. If there is a way to revert it without any manual intervention, I'm all ears. For now, I've tried numerous "system" things to get it to the state you want, and nothing's worked.

Reply:
This typically happens if on different versions of `less-plugin-autoprefix`